api/nakadi-event-bus-api.yaml

```diff
@@ -888,6 +888,7 @@ definitions:
               Timestamp of creation of the Event generated by the producer.
             type: string
             format: date-time
+            example: '1996-12-19T16:39:57-08:00'
```

Comment:
I would use the +01:00 for the time zone 😀

Apropos, does API guild allow non-UTF times?